config/config.go

```diff
@@ -167,6 +168,16 @@ func newDefaultMetricsConfig() *MetricsConfig {
 	}
 }
 
+type MessagePoolConfig struct {
+	Limit int `json:"limit"`
+}
+
+func newDefaultMessagePoolConfig() *MessagePoolConfig {
+	return &MessagePoolConfig{
+		Limit: 100,
```

Comment:
I think we can start larger than this, like a few thousand. Perhaps find the max message size and then set this to allow 256MB of messages (and document that calculation here).

Reply:
Max object sizes are 1MiB<sub>[0](https://github.com/filecoin-project/specs/blob/master/data-structures.md#encoding-considerations)</sub> (2<sup>20</sup>), that would put us to ~256 messages.